coordinator/src/bin/coordinator.rs

```diff
@@ -218,8 +227,23 @@ async fn main() -> Result<()> {
                             }
                         };
 
+                    // Upon collab closing an expired position we cannot charge a fee using an
+                    // invoice. This dummy hash exists in the database to
+                    // represent zero-amount invoices.
+                    let zero_amount_payment_hash_dummy = PaymentHash(
+                        <[u8; 32]>::from_hex(
+                            "6f9b8c95c2ba7b1857b19f975372308161fedf50feb78a252200135a41875210",
```

Comment:
🔧 I think it would help if this is private to the `db` module. We could have a `close_expired_position` method without the `fee_payment_hash`. And maybe a `NewTrade::close_expired` encapsulating this stuff.

Reply:
If we agree that the trade fee *should* always be set (but it is not for this particular case at the moment because it's not implemented) then I would keep this in the business logic. The database expects the payment hash to be set and should not "default" it (initial migration excepted, but that's a different case).